### DISCUSSION TITLE
Expand length of the QTH field in the database

### DISCUSSION THREAD

**Discussion by R1BLH** [wavelog/wavelog#511] [Ideas]
It would be good to extend the COL_QTH column of the TABLE_HRD_CONTACTS_V01 table to 128 symbols.
I have already found 5 records that are 64 symbols long. This means that the original content could be truncated.

![изображение](https://github.com/wavelog/wavelog/assets/16667173/f0743626-6490-449f-8186-039b55970e31)

**Comment by HB9HIL** [comment]
@int2001 

I don't see a problem in expanding this. What do you think? We don't have to do an extra mig for that. We could include this in the next release mig or any other.

**Reply by int2001** [reply]
Okay for me. Can't spot a problem here. Checked ADIF. It says "string" (no limit given).

**Comment by HB9HIL** [comment]
@R1BLH 

With the release of Version 1.7 I sneaked in the change for this column. The column COL_QTH can now be 128 characters long :)

**Reply by R1BLH** [reply]
Thank you!
